src/modules/vtol_att_control/vtol_att_control_params.c

```diff
@@ -191,6 +190,8 @@ PARAM_DEFINE_FLOAT(VT_TRANS_TIMEOUT, 15.0f);
  * @unit s
  * @min 0.0
  * @max 20.0
+ * @increment 0.1
+ * @decimal 1
```

Comment:
Maybe a detail but wouldn't it be nice to keep the order consistent?
```suggestion
 * @decimal 1
 * @increment 0.1
```

Reply:
I copy pasted it from somewhere in the same file, but agree this should be the order to strive for.